src/containers/__snapshots__/Permissions.spec.jsx.snap

```diff
@@ -32,14 +32,41 @@ exports[`Permissions should display appName when query has been loaded 1`] = `
               />
             </div>
             <div
-              class="MuiListItemText-root"
+              data-primary="Alan"
+              data-secondary="0 permissions"
```

Comment:
would that be possible to check in one test, when there are permissions? (instead of  0 permissions)